src/System.IO.FileSystem/tests/PortedCommon/IOInputs.cs

```diff
@@ -18,10 +18,20 @@ internal static class IOInputs
     public static bool CaseSensitive { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) | RuntimeInformation.IsOSPlatform(OSPlatform.OSX); } }
     public static bool CaseInsensitive { get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) | RuntimeInformation.IsOSPlatform(OSPlatform.OSX); } }
 
-    // Unix values vary system to system; just using really long values here likely to be more than on the average system
-    public static readonly int MaxDirectory = 247; // Does not include trailing \0. This the maximum length that can be passed to APIs taking directory names, such as Directory.CreateDirectory, Directory.Move
-    public static readonly int MaxPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 259 : 10000;      // Does not include trailing \0.
-    public static readonly int MaxComponent = 255;
+    // Max path length (minus trailing \0). Unix values vary system to system; just using really long values here likely to be more than on the average system.
+    public static readonly int MaxPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 259 : 10000;
```

Comment:
> Doesn't the current Unix implementation fail with any Unix paths longer than the Windows MAX_PATH or has that been fixed already?

That should have been fixed several weeks ago.  If you find anything to the contrary, please shout.  The only such issues that are known are those that hit the Path implementation in mscorlib and Path stuff in the libcoreclr PAL, and those should be restricted to support implemented in mscorlib and the runtime, e.g. assembly loading, resource loading, etc.